Week-5/symbols/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Unique Transaction IDs</title>
+    <script src="script.js" defer></script>
+</head>
+<body>
+    <h1>Unique Transaction IDs</h1>
```

Comment:
`The usage of <h1> and <p> tags provides a clear, user-friendly message about what the user should expect. It might be helpful to also display the unique IDs on the webpage itself, not just the console, to make it more interactive.`